contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -18,12 +18,13 @@ At a very high level, the process to contribute and improve is pretty simple:
 - Submit your Pull Request
 
 The following sections describe some guidelines that can come in handy with the above process. 
-*Followed by the guidelines, is a cheat sheet with frequently used git commands.*
+*Followed by the guidelines, is a [cheatsheet](./contribute/git-cheatsheet.md) with frequently used git commands.*
```

Comment:
Git commands.

Reply:
I would prefer for it to be smallcase. Since the official site uses it as the same if it is not violating the grammatical sense.